src/utils/languages.ts

```diff
@@ -0,0 +1,24 @@
+export interface Language {
+    value: string;
+    label: string;
+    hasSymbol?: string;
```

Comment:
That unused arg warning also brought t my attention that we don't seem to really need `hasSymbol`. unlike currency I don't think the symbol is super import for identifying the current language like it is for currency

Reply:
Good catch! I left it there since the beginning, removing it.